build/css-selector-generator.js

```diff
@@ -4,7 +4,14 @@
 
   CssSelectorGenerator = (function() {
     CssSelectorGenerator.prototype.default_options = {
-      selectors: ['id', 'class', 'tag', 'nthchild']
+      selectors: ['id', 'class', 'tag', 'nthchild'],
+      prefix_tag: false,
+      log: false,
+      attribute_blacklist: [],
+      attribute_whitelist: [],
+      quote_attribute_when_needed: false,
+      id_blacklist: [],
+      class_blacklist: []
```

Comment:
If I can make a request to go further, I would like to have `class_whitelist` (lots of classes in my application are not reliable). I guess that could be the same for classes. 

That could be done in another PR (I can try to submit another one when this one is merged).

Thanks!

Reply:
The idea is to have a small script that looks at a page and automatically blacklists the classes that are present more than x%. The way I am looking at the whitelist is as a prioritized list of items. Basically, ignore everything in blacklist, consider whitelisted items first and rest later. I can add support for class and id whitelist.

Reply:
>  Basically, ignore everything in blacklist, consider whitelisted items first and rest later.

Well, I would need a whitelist of attributes and blacklist everything else. Do you think that could be feasible?

> I can add support for class and id whitelist.

I would be grateful!

Florent